client.ts

```diff
@@ -24,6 +25,21 @@ export class Client {
     return await this._connection.query(query);
   }
 
+  async transaction(
+    text: string | string[],
+    args?: unknown[],
+    name?: string,
```

Comment:
This argument is unused

Reply:
yeah, I used the `QueryConfig` interface for the arguments. I added a new commit where I removed it, let me know if this is preffered